OxfordBodleian/Juel-Jensen/BDLaethg36.xml

```diff
@@ -102,90 +95,304 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                but once in the upper margin with a <foreign xml:lang="gez">tamallas</foreign> sign in the text 
+                                (<locus target="#45r"/>).
+                                There are different type of deletions: erasures (e.g. <locus target="#1r #5v #38v"/>), 
+                                erasures with subsequent addition of line fillers 
+                                (e.g. <locus target="#13r #25v"/>),
+                                marking with over- and underlining (e.g. <locus target="#4r #20r #32r"/>),
+                                marking with over- and underlining with subsequent erasure (e.g. <locus target="#20v #28r #30r"/>),
+                                blotting out (e.g. <locus target="#12r #13r #25v"/>).
+                            </item>
+                        </list>
+                    </additions>
+                    
+                </physDesc>
+
+                    </msPart>
 
-                    <additional>
-                        <adminInfo>
-                            <recordHist>
-                                <source>
-                                  <!--  <listBibl type="secondary">
-                                        <bibl></bibl>
-                                    </listBibl>-->
-                                </source>
-                            </recordHist>
-                        </adminInfo>
-                    </additional>
+<msPart xml:id="p2">
+    <msIdentifier></msIdentifier>
+    
+    <msContents>
+        <summary/>
+        
+        <msItem xml:id="p2_i1">
+            <locus target="#53ra"/>
+            <title type="incomplete">Beginning of a list of the commemoration dates of the apostles</title>
+            <note>The text is reproduced in full below.</note>
+            <incipit xml:lang="gez">
+            ሠረቀ፡ መስከረም፡ በ፩በርተሎሜዎስ፡ ሠረቀ፡ ጥቅምት፡ በ፲ወ፪ማቴዎስ፡ በ፳ወ፪ሉቃስ፡ ሠ፡ ኅ
+            </incipit>
+        </msItem>
+    </msContents>
+    
+    <physDesc>
+        <objectDesc form="Leaf">
+            
+            <supportDesc>
+                <support>
+                    <material key="parchment"/>
+                </support>
+            </supportDesc>
+            
+            <layoutDesc>
+                <layout columns="1" writtenLines="5" ruledLines="11">
+                    <locus target="#53r"></locus>
+                    <note>Five lines of text in the upper part of the folio.</note>
+                    <ab type="pricking">Ruling and pricking are only partly visible.</ab>
+                </layout>
+            </layoutDesc>
+            
+        </objectDesc>
+        
+        <handDesc>
+            <handNote xml:id="h2" script="Ethiopic">
+                <locus target="#53r"/>
+                <seg type="ink">Black.</seg>
+                <date notBefore="1850" notAfter="1950"></date>
+                <desc>
+                    Irregular hand, but characters are regularly spaced.
```

Comment:
Is it a different hand from h1? (I ask because the description is identical :))

Reply:
It is different, but I'm no artist when it comes to these descriptions :D